docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: "3"
+services:
+    app:
+        build:
+            context: .
+        ports:
+            - "8000:8000"
+        volumes:
+            - .:/app
+        command: >
+            sh -c " python manage.py wait_for_db &&
+                    python manage.py migrate &&
+                    python manage.py runserver 0.0.0.0:8000"
+        environment:
+            - PYTHONUNBUFFERED=1
+            - POSTGRES_DB=db
+            - POSTGRES_USER=postgres
+            - POSTGRES_PASSWORD=asdf!qwe
+            - HOST_DB=db
+        depends_on:
+            - db
+
+    db:
+        image: postgres:14-alpine
+        ports:
+            - "5433:5432"
+        environment:
+            - POSTGRES_DB=db
+            - POSTGRES_USER=postgres
+            - POSTGRES_PASSWORD=asdf!qwe

```

Comment:
It is crucial to handle credentials securely in your Dockerfile and not expose them directly. To achieve this, you can create an environment (env) file to store your sensitive credentials and add it to your .gitignore file to prevent it from being committed to your version control